README.md

```diff
@@ -12,9 +12,11 @@ Spyder Plugin for executing Vim commands inside the code editor. We currently su
 ## Installation
 To install this plugin, you can use either ``pip`` or ``conda`` package managers, as it follows:
 
-Using pip:
+Build from source:
 ```
-pip install spyder-vim
+git clone https://github.com/spyder-ide/spyder-vim
+cd spyder-vim
+python setup.py install .
```

Comment:
I think this should be `pip install .` instead. @andfoy, what do you way?

Reply:
I always prefer using ``pip install .``. However, these instructions shouldn't be under the Installation section

Reply:
Yep, sure.